server/app.js

```diff
@@ -18,6 +19,7 @@ app.use(express.static(join(__dirname, "public")));
 
 app.use("/", indexRouter);
 app.use("/ping", pingRouter);
+app.use("/register", register);
```

Comment:
I feel like we would have register and login in the same file, so maybe you want to create a router for the signup routes only. Also, register doesn't fit the naming convention of nameRouter

Reply:
So something like this? 
app.use("/register", authRouter.register);
app.use("/login, authRouter.login);

Reply:
Maybe instead create a usersRouter, and have the route be /users/login, etc.?